packages/mdc-list/README.md

```diff
@@ -1,49 +1,23 @@
-<!--docs:
-title: "Lists"
-layout: detail
-section: components
-excerpt: "Lists present multiple line items vertically as a single continuous element."
-iconId: list
-path: /catalog/lists/
--->
-
 # Lists
 
-<!--<div class="article__asset">
-  <a class="article__asset-link"
-     href="https://material-components-web.appspot.com/list.html">
-    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/lists.png" width="365" alt="Lists screenshot">
-  </a>
-</div>-->
-
-MDC List provides styles which implement [Material Design Lists](https://material.io/guidelines/components/lists.html) -
-"A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are
-supported (with three-line lists [planned](https://github.com/material-components/material-components-web/issues/31)).
-MDC Lists are designed to be accessible and RTL aware.
+MDC List provides styles which implement Material Design Lists - "A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are supported (with three-line lists planned). MDC Lists are designed to be accessible and RTL aware.
 
 ## Design & API Documentation
 
-<ul class="icon-list">
-  <li class="icon-list-item icon-list-item--spec">
-    <a href="https://material.io/guidelines/components/lists.html">Material Design guidelines: Lists</a>
-  </li>
-  <li class="icon-list-item icon-list-item--link">
-    <a href="https://material-components-web.appspot.com/list.html">Demo</a>
-  </li>
-</ul>
+* [Material Design guidelines: List](https://material.io/guidelines/components/lists.html)
```

Comment:
This should also follow the [template](https://github.com/material-components/material-components-web/blob/master/docs/code/readme_template.md)